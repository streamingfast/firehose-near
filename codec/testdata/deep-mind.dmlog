DMLOG BLOCK 21 71b52c9ac404ed7f0666d94f77069dfe4c4fbb478d06cec77e73bf21295c82db 0a056e6f64653312ff0408151a220a20000000000000000000000000000000000000000000000000000000000000000022220a2081b08f80a7c38ef04e4678c51d10312fde4077674319ada99ad79736dd80c1ad2a220a2071b52c9ac404ed7f0666d94f77069dfe4c4fbb478d06cec77e73bf21295c82db32220a20f96184f79bd4a90daf4d6d09c8ccf4dc47d1123bae3d4df8c9b899921d797d0a3a220a207c133a1b1060edd83d44965b2b6372078ce31b1903689943c1792c186ace52ab42220a206637dbc9de3e1d5a5e9831bce264f6ee817b6fff3a04319fc3930e33c2f67d664a220a20455f16820a1f22802dd5d85128e7e95f7e6b67fd292b31aaf897f7c02836433552220a2066687aadf862bd776c8fc18b8e9f8e20089714856ee233b3902a591d0d5f29255a220a20616c1e95a59742d845dae42a360b9b458a69517d60676535112b78a2c1d3e45a60016a220a20000000000000000000000000000000000000000000000000000000000000000070f7b4a0fafd9ce1d51678f7b4a0fafd9ce1d5168201220a20dc736da2a63d1081c3353507d52135dd2574dd0ab134e54a6f069f314d0852fa920101019a01120a100000000000000000000000003b9aca00aa01120a1000000000000000000000000000000000b201120a100000f8fbf60d6476dabaaafe1dbc1800c201220a201cccd8857c34243a535aea02eed8614658db62f1d100ad35f5685abde3a595cbca01220a20f96184f79bd4a90daf4d6d09c8ccf4dc47d1123bae3d4df8c9b899921d797d0ad201220a2047dc1272133419a48346634a0b27e848b0e52443bd231667ce3ded2ca788ba83da01220a2095bd3a2a4e4c457c5f718f012442e71d2fca80f361540d7557265d7dcb289527f8012f1aad020a205f03faba59688a1b928025852b3b838c86c922ec40ef94a2b3813a50bfebf4101220f96184f79bd4a90daf4d6d09c8ccf4dc47d1123bae3d4df8c9b899921d797d0a1a20547c0d836d97e20e0122c2fc0bbc01d1170886bf8e9cfba75662cf7fa156377322208421876605fa71437bb89798a55f30c072b5ae56c228bf2dbe05d955f966f1512a205124899990e714520a5c594d8cef076f35515747cb5f4f823492e5947bf1f94c30c7013815401550e4c9d0cbad0c5880809aa6eaafe30162120a10000000000000000000000000000000006a120a10000000000000001703e293f46243e80072209dd117f2aaec82330d1143b23b51239340715734377ae4d8949ecae055eda8b27a20000000000000000000000000000000000000000000000000000000000000000022f4061281040a056e6f64653312ad020a205f03faba59688a1b928025852b3b838c86c922ec40ef94a2b3813a50bfebf4101220f96184f79bd4a90daf4d6d09c8ccf4dc47d1123bae3d4df8c9b899921d797d0a1a20547c0d836d97e20e0122c2fc0bbc01d1170886bf8e9cfba75662cf7fa156377322208421876605fa71437bb89798a55f30c072b5ae56c228bf2dbe05d955f966f1512a205124899990e714520a5c594d8cef076f35515747cb5f4f823492e5947bf1f94c30c7013815401550e4c9d0cbad0c5880809aa6eaafe30162120a10000000000000000000000000000000006a120a10000000000000001703e293f46243e80072209dd117f2aaec82330d1143b23b51239340715734377ae4d8949ecae055eda8b27a20000000000000000000000000000000000000000000000000000000000000000022c7010a046e6561721210626174746c656669656c642e6e6561721a220a201374e6d468549ba8cb07c35f2d616a1846b4ccc420d8805c56fc9e1893b2f5ea5288010a046e65617212220a203d559878f03d5279f189aaaf77898ac6b91499ec5cd5a60e3f9059a1122e19ff1a120a100000000000000000000000003d648d8032020a00321622140a120a1000000000019d971e4fe8401e74000000322c322a0a220a203d559878f03d5279f189aaaf77898ac6b91499ec5cd5a60e3f9059a1122e19ff1204120212001aed020aa0010a0012220a2071b52c9ac404ed7f0666d94f77069dfe4c4fbb478d06cec77e73bf21295c82db1a220a201374e6d468549ba8cb07c35f2d616a1846b4ccc420d8805c56fc9e1893b2f5ea225412220a20022a186964b37c278e40a6a3c199168dcdc36d0112c757807593f93543c69f3518e4c9d0cbad0c22120a10000000000000001703e293f46243e8002a10626174746c656669656c642e6e656172b2010012c7010a046e6561721210626174746c656669656c642e6e6561721a220a201374e6d468549ba8cb07c35f2d616a1846b4ccc420d8805c56fc9e1893b2f5ea5288010a046e65617212220a203d559878f03d5279f189aaaf77898ac6b91499ec5cd5a60e3f9059a1122e19ff1a120a100000000000000000000000003d648d8032020a00321622140a120a1000000000019d971e4fe8401e74000000322c322a0a220a203d559878f03d5279f189aaaf77898ac6b91499ec5cd5a60e3f9059a1122e19ff120412021200
DMLOG BLOCK 309 ca4307191bd9efc54b9e62ca02fc62cd4bce6565a167ddb3d9d4458869c9588c 0a056e6f64653012800508b5021a220a20585592bf83374131022d29fa4be7106748c9c9864d010d7497bf7c19e7da556922220a204da42e2d3a18703603ab8d02baab46d7d9ed31950cd4194457c6f578af98b8042a220a20ca4307191bd9efc54b9e62ca02fc62cd4bce6565a167ddb3d9d4458869c9588c32220a206f0765c2d5272c0701e9c9f2db1e607a9c6ce17c34af55d835deb8dda7b155143a220a204b155c41506ca7b7ab39892dc9afa3ee9729cf70e2e32ee2f23b3a4ddd546a8742220a207a4fde36a426deaf5e1252f5718a06e1787f8d12cb7da5a18057fa9ffb66ce0d4a220a20a7544a10cedbb8a631cf6e331b984f499aa6de4549ea82026ba256b50f550b5352220a20b72f8a7d580c83a78f1af5065b0fed9d86b6a982020365faba00390d712bc0af5a220a2066687aadf862bd776c8fc18b8e9f8e20089714856ee233b3902a591d0d5f292560016a220a20000000000000000000000000000000000000000000000000000000000000000070dee9cff6c89ee1d51678dee9cff6c89ee1d5168201220a20c7685d2ff5c853c2223f6c7afb4635d3224bc368fa782fa09551d527a59da7cf920101019a01120a100000000000000000000000003b9aca00aa01120a1000000000000000000000000000000000b201120a100000fb606de10890bd061d1561aa5130c201220a209bd95b431f26883cdd1da2d99c8d8e62b468b61609295138ec279234ddeb6ab8ca01220a206f0765c2d5272c0701e9c9f2db1e607a9c6ce17c34af55d835deb8dda7b15514d201220a20ffae6b8eba888c7da8cc015b40dc3b3dbb97bd200e4e6c43db15622f78131c78da01220a20e0c8d8185401afcab5595c2e2d626c5b886287484424e502fa0331226685ff8bf8012f1aa8020a20fc60ca9d56f1d5bf258a61f701b2e4ed93948d13113e86c480cefd527d7dcf7312206f0765c2d5272c0701e9c9f2db1e607a9c6ce17c34af55d835deb8dda7b155141a200000000000000000000000000000000000000000000000000000000000000000222035e8c48d619cc1e18845000bb24e433da0b5ceef8c7af921b76080229ef65fe52a200b836eec4e44e684225caeb50413e9b916d178f0a9f5b0c09e98ad78fa94e4ce30bb0138b50240b5025880809aa6eaafe30162120a10000000000000000000000000000000006a120a10000000000000000000000000000000007220ee9b92e324e0341a965daf29b39555030e2c0e5591d7d3e2bd05ff785f946b057a20ed8f24e2865c40015ad607d5d4598414435dcf65a064c6d1957db1a46614865622f10412ee040a056e6f64653012a8020a20fc60ca9d56f1d5bf258a61f701b2e4ed93948d13113e86c480cefd527d7dcf7312206f0765c2d5272c0701e9c9f2db1e607a9c6ce17c34af55d835deb8dda7b155141a200000000000000000000000000000000000000000000000000000000000000000222035e8c48d619cc1e18845000bb24e433da0b5ceef8c7af921b76080229ef65fe52a200b836eec4e44e684225caeb50413e9b916d178f0a9f5b0c09e98ad78fa94e4ce30bb0138b50240b5025880809aa6eaafe30162120a10000000000000000000000000000000006a120a10000000000000000000000000000000007220ee9b92e324e0341a965daf29b39555030e2c0e5591d7d3e2bd05ff785f946b057a20ed8f24e2865c40015ad607d5d4598414435dcf65a064c6d1957db1a4661486561ab9020a750a08626f622e6e65617212220a203d559878f03d5279f189aaaf77898ac6b91499ec5cd5a60e3f9059a1122e19ff1890feac0d22046e6561722a1622140a120a1000000000000000056bc75e2d631000003a220a206b077438c7f1b08230f61200e18db4bdcb234c7a89f97ac3e210931746ef848212bf010abc010a0012220a20ca4307191bd9efc54b9e62ca02fc62cd4bce6565a167ddb3d9d4458869c9588c1a220a206b077438c7f1b08230f61200e18db4bdcb234c7a89f97ac3e210931746ef8482227012220a2091ab7f5e35e50475b673b06ea63979de7f6510187197ea419a86b622d633a6ca18c4c9deb5bf0622120a10000000000000000c1947be1f31eaa8002a08626f622e6e656172ba01240a220a2091ab7f5e35e50475b673b06ea63979de7f6510187197ea419a86b622d633a6ca