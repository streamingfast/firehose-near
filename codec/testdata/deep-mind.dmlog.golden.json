[{
  "author": "node0",
  "header": {
    "height": 60,
    "epoch_id": {
      "bytes": "AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA="
    },
    "next_epoch_id": {
      "bytes": "gbCPgKfDjvBORnjFHRAxL95Ad2dDGa2pmteXNt2Awa0="
    },
    "hash": {
      "bytes": "kb1Yjr0pvaOl9dCcnJtj/N+dLrAdhDltlqpx2wuDnMk="
    },
    "prev_hash": {
      "bytes": "B8in2uftjph1encPlU2de9Ckio3c5bN2248BwbeunFU="
    },
    "prev_state_root": {
      "bytes": "TTnCSZc24oDBVek+y7Myz4j+0B2FMcEPWWMAouOwWr0="
    },
    "chunk_receipts_root": {
      "bytes": "ek/eNqQm3q9eElL1cYoG4Xh/jRLLfaWhgFf6n/tmzg0="
    },
    "chunk_headers_root": {
      "bytes": "H3ldE7qd9BLvYSci1JcFOo6+KOh6m60ykqA9v8Ll4MM="
    },
    "chunk_tx_root": {
      "bytes": "AkvBJMWx4AVyVIBMN0+nUnp4qfxAZhVNRMP5byNHeYI="
    },
    "outcome_root": {
      "bytes": "Zmh6rfhivXdsj8GLjp+OIAiXFIVu4jOzkCpZHQ1fKSU="
    },
    "chunks_included": 1,
    "challenges_root": {
      "bytes": "AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA="
    },
    "timestamp": 1633610762794997973,
    "timestamp_nanosec": 1633610762794997973,
    "random_value": {
      "bytes": "4k/RxSq2oFPe5v1zNMuQHDmQ6gOh42mBbxpAoLg7VNg="
    },
    "chunk_mask": [
      true
    ],
    "gas_price": {
      "bytes": "AAAAAAAAAAAAAAAAO5rKAA=="
    },
    "rent_paid": {
      "bytes": "AAAAAAAAAAAAAAAAAAAAAA=="
    },
    "validator_reward": {
      "bytes": "AAAAAAAAAAAAAAAAAAAAAA=="
    },
    "total_supply": {
      "bytes": "AAD4+/YNWY8m/+Yx8h3yAA=="
    },
    "last_final_block": {
      "bytes": "B8in2uftjph1encPlU2de9Ckio3c5bN2248BwbeunFU="
    },
    "last_ds_final_block": {
      "bytes": "R9wSchM0GaSDRmNKCyfoSLDlJEO9IxZnzj3tLKeIuoM="
    },
    "next_bp_hash": {
      "bytes": "Sl5HKUKGnJ71lRRbYx13DDQmvIWC79LaAsKBavV6woU="
    }
  },
  "chunk_headers": [
    {
      "chunk_hash": "nT8PhaoQsZ7ksoJ2ETEVSCTf7xfn4qPBJJoFGRoLGyo=",
      "prev_block_hash": "B8in2uftjph1encPlU2de9Ckio3c5bN2248BwbeunFU=",
      "outcome_root": "AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA=",
      "prev_state_root": "Hr9jZCfGXY8kOq8GzOhxx39RsfxLupDh3USd2VhVLcY=",
      "encoded_merkle_root": "kkXfma82mtW0nlLCBZZQuziF5franL5To1xOSuHv2vk=",
      "encoded_length": 187,
      "height_created": 60,
      "height_included": 60,
      "gas_limit": 1000000000000000,
      "validator_reward": {
        "bytes": "AAAAAAAAAAAAAAAAAAAAAA=="
      },
      "balance_burnt": {
        "bytes": "AAAAAAAAAAAAAAAAAAAAAA=="
      },
      "outgoing_receipts_root": "7puS4yTgNBqWXa8ps5VVAw4sDlWR19PivQX/eF+UawU=",
      "tx_root": "5365GiebBgRgvk4l9sCk+Hl6wn1I41icqtsSrcWx00o="
    }
  ],
  "shards": [
    {
      "chunk": {
        "author": "node0",
        "header": {
          "chunk_hash": "nT8PhaoQsZ7ksoJ2ETEVSCTf7xfn4qPBJJoFGRoLGyo=",
          "prev_block_hash": "B8in2uftjph1encPlU2de9Ckio3c5bN2248BwbeunFU=",
          "outcome_root": "AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA=",
          "prev_state_root": "Hr9jZCfGXY8kOq8GzOhxx39RsfxLupDh3USd2VhVLcY=",
          "encoded_merkle_root": "kkXfma82mtW0nlLCBZZQuziF5franL5To1xOSuHv2vk=",
          "encoded_length": 187,
          "height_created": 60,
          "height_included": 60,
          "gas_limit": 1000000000000000,
          "validator_reward": {
            "bytes": "AAAAAAAAAAAAAAAAAAAAAA=="
          },
          "balance_burnt": {
            "bytes": "AAAAAAAAAAAAAAAAAAAAAA=="
          },
          "outgoing_receipts_root": "7puS4yTgNBqWXa8ps5VVAw4sDlWR19PivQX/eF+UawU=",
          "tx_root": "5365GiebBgRgvk4l9sCk+Hl6wn1I41icqtsSrcWx00o="
        },
        "transactions": [
          {
            "transaction": {
              "signer_id": "bob.near",
              "public_key": {
                "bytes": "PVWYePA9Unnxiaqvd4mKxrkUmexc1aYOP5BZoRIuGf8="
              },
              "nonce": 28000002,
              "receiver_id": "near",
              "actions": [
                {
                  "Action": {
                    "Transfer": {
                      "deposit": {
                        "bytes": "AAAAAAAAAAVrx14tYxAAAA=="
                      }
                    }
                  }
                }
              ],
              "hash": {
                "bytes": "fwGRrpMS/0sf9KK8VqsH4QMDJfYmad/3a2tjVqFmqik="
              }
            },
            "outcome": {
              "execution_outcome": {
                "proof": {},
                "block_hash": {
                  "bytes": "kb1Yjr0pvaOl9dCcnJtj/N+dLrAdhDltlqpx2wuDnMk="
                },
                "id": {
                  "bytes": "fwGRrpMS/0sf9KK8VqsH4QMDJfYmad/3a2tjVqFmqik="
                },
                "outcome": {
                  "receipt_ids": [
                    {
                      "bytes": "R5IAyIaJjm1kpaY7g6mesnfFbWw3k1rhJZMHehRwRKs="
                    }
                  ],
                  "gas_burnt": 223182562500,
                  "tokens_burnt": {
                    "bytes": "AAAAAAAAAAwZR74fMeqoAA=="
                  },
                  "executor_id": "bob.near",
                  "Status": {
                    "SuccessReceiptId": {
                      "id": {
                        "bytes": "R5IAyIaJjm1kpaY7g6mesnfFbWw3k1rhJZMHehRwRKs="
                      }
                    }
                  }
                }
              }
            }
          }
        ]
      }
    }
  ]
}
,{
  "author": "node3",
  "header": {
    "height": 61,
    "epoch_id": {
      "bytes": "AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA="
    },
    "next_epoch_id": {
      "bytes": "gbCPgKfDjvBORnjFHRAxL95Ad2dDGa2pmteXNt2Awa0="
    },
    "hash": {
      "bytes": "X3o0kf3wZofH5nTswnI2m0hp3wkEHlmd+ymAjOuBGPA="
    },
    "prev_hash": {
      "bytes": "kb1Yjr0pvaOl9dCcnJtj/N+dLrAdhDltlqpx2wuDnMk="
    },
    "prev_state_root": {
      "bytes": "avgw3eyq2APZ3ltndO18XHW3C0gj+uoyfkbk/J9NLAo="
    },
    "chunk_receipts_root": {
      "bytes": "L+J93s2RIYvtebbPoP01/Ub+vVL0xUeiK2iUB2MqF80="
    },
    "chunk_headers_root": {
      "bytes": "anlCRzzLvxbYzZ8R5oAiO7f2kZVjEDQVKyAR23C/ezc="
    },
    "chunk_tx_root": {
      "bytes": "Zmh6rfhivXdsj8GLjp+OIAiXFIVu4jOzkCpZHQ1fKSU="
    },
    "outcome_root": {
      "bytes": "ZkzqcSUqFmj/oXu1wJ57aOTDd8kslf6eyrQwEom8FBQ="
    },
    "chunks_included": 1,
    "challenges_root": {
      "bytes": "AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA="
    },
    "timestamp": 1633610762996590346,
    "timestamp_nanosec": 1633610762996590346,
    "random_value": {
      "bytes": "ec9QivF0/twOWamWKOIqpRoOiXWWwvAl+tN0OLteHRo="
    },
    "chunk_mask": [
      true
    ],
    "gas_price": {
      "bytes": "AAAAAAAAAAAAAAAAO5rKAA=="
    },
    "rent_paid": {
      "bytes": "AAAAAAAAAAAAAAAAAAAAAA=="
    },
    "validator_reward": {
      "bytes": "AAAAAAAAAAAAAAAAAAAAAA=="
    },
    "total_supply": {
      "bytes": "AAD4+/YNWYMNuCgSwDNKAA=="
    },
    "last_final_block": {
      "bytes": "kb1Yjr0pvaOl9dCcnJtj/N+dLrAdhDltlqpx2wuDnMk="
    },
    "last_ds_final_block": {
      "bytes": "R9wSchM0GaSDRmNKCyfoSLDlJEO9IxZnzj3tLKeIuoM="
    },
    "next_bp_hash": {
      "bytes": "dSiJr3sl3kWupAR3k9RIIHQNdWnHNQ7vOtpDNFi8hj4="
    }
  },
  "chunk_headers": [
    {
      "chunk_hash": "+XmRhmfzlYA5rfjpp4mSPmvkb6QDg/uqitArLCYzX3A=",
      "prev_block_hash": "kb1Yjr0pvaOl9dCcnJtj/N+dLrAdhDltlqpx2wuDnMk=",
      "outcome_root": "ezCLaCs5UIRsehWWLYjh/HqpGwmHOQE5vW+22j7vZL0=",
      "prev_state_root": "zQMqMBEor7Sw8x+1cmOi5/v3BKaYhZ7aZ5Tvro37dU0=",
      "encoded_merkle_root": "XQgciT6CyH14JJdEMHTjOz9IEEEuMjo9CzSrUm6JrmU=",
      "encoded_length": 151,
      "height_created": 61,
      "height_included": 61,
      "gas_used": 223182562500,
      "gas_limit": 1000000000000000,
      "validator_reward": {
        "bytes": "AAAAAAAAAAAAAAAAAAAAAA=="
      },
      "balance_burnt": {
        "bytes": "AAAAAAAAAAwZR74fMeqoAA=="
      },
      "outgoing_receipts_root": "1VIGJ0l+PtbIZl97l/7SqM/mrJ5IFH9keqC/a9FpiNA=",
      "tx_root": "AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA="
    }
  ],
  "shards": [
    {
      "chunk": {
        "author": "node3",
        "header": {
          "chunk_hash": "+XmRhmfzlYA5rfjpp4mSPmvkb6QDg/uqitArLCYzX3A=",
          "prev_block_hash": "kb1Yjr0pvaOl9dCcnJtj/N+dLrAdhDltlqpx2wuDnMk=",
          "outcome_root": "ezCLaCs5UIRsehWWLYjh/HqpGwmHOQE5vW+22j7vZL0=",
          "prev_state_root": "zQMqMBEor7Sw8x+1cmOi5/v3BKaYhZ7aZ5Tvro37dU0=",
          "encoded_merkle_root": "XQgciT6CyH14JJdEMHTjOz9IEEEuMjo9CzSrUm6JrmU=",
          "encoded_length": 151,
          "height_created": 61,
          "height_included": 61,
          "gas_used": 223182562500,
          "gas_limit": 1000000000000000,
          "validator_reward": {
            "bytes": "AAAAAAAAAAAAAAAAAAAAAA=="
          },
          "balance_burnt": {
            "bytes": "AAAAAAAAAAwZR74fMeqoAA=="
          },
          "outgoing_receipts_root": "1VIGJ0l+PtbIZl97l/7SqM/mrJ5IFH9keqC/a9FpiNA=",
          "tx_root": "AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA="
        },
        "receipts": [
          {
            "predecessor_id": "bob.near",
            "receiver_id": "near",
            "receipt_id": {
              "bytes": "R5IAyIaJjm1kpaY7g6mesnfFbWw3k1rhJZMHehRwRKs="
            },
            "Receipt": {
              "Action": {
                "signer_id": "bob.near",
                "signer_public_key": {
                  "bytes": "PVWYePA9Unnxiaqvd4mKxrkUmexc1aYOP5BZoRIuGf8="
                },
                "gas_price": {
                  "bytes": "AAAAAAAAAAAAAAAAPWSNgA=="
                },
                "actions": [
                  {
                    "Action": {
                      "Transfer": {
                        "deposit": {
                          "bytes": "AAAAAAAAAAVrx14tYxAAAA=="
                        }
                      }
                    }
                  }
                ]
              }
            }
          }
        ]
      },
      "receipt_execution_outcomes": [
        {
          "execution_outcome": {
            "proof": {},
            "block_hash": {
              "bytes": "X3o0kf3wZofH5nTswnI2m0hp3wkEHlmd+ymAjOuBGPA="
            },
            "id": {
              "bytes": "R5IAyIaJjm1kpaY7g6mesnfFbWw3k1rhJZMHehRwRKs="
            },
            "outcome": {
              "receipt_ids": [
                {
                  "bytes": "ycJPBWB6sfpiSIqa6CuTjba+9D31BOApIeZ7aMidcm0="
                }
              ],
              "gas_burnt": 223182562500,
              "tokens_burnt": {
                "bytes": "AAAAAAAAAAwZR74fMeqoAA=="
              },
              "executor_id": "near",
              "Status": {
                "SuccessValue": {}
              }
            }
          },
          "receipt": {
            "predecessor_id": "bob.near",
            "receiver_id": "near",
            "receipt_id": {
              "bytes": "R5IAyIaJjm1kpaY7g6mesnfFbWw3k1rhJZMHehRwRKs="
            },
            "Receipt": {
              "Action": {
                "signer_id": "bob.near",
                "signer_public_key": {
                  "bytes": "PVWYePA9Unnxiaqvd4mKxrkUmexc1aYOP5BZoRIuGf8="
                },
                "gas_price": {
                  "bytes": "AAAAAAAAAAAAAAAAPWSNgA=="
                },
                "actions": [
                  {
                    "Action": {
                      "Transfer": {
                        "deposit": {
                          "bytes": "AAAAAAAAAAVrx14tYxAAAA=="
                        }
                      }
                    }
                  }
                ]
              }
            }
          }
        }
      ]
    }
  ]
}
]